coordinator/statement_executor.go

```diff
@@ -758,6 +774,19 @@ func (e *StatementExecutor) executeShowShardsStatement(stmt *influxql.ShowShards
 	return rows, nil
 }
 
+func (e *StatementExecutor) executeShowSeriesCardinalityStatement(stmt *influxql.ShowSeriesCardinalityStatement) (models.Rows, error) {
+	n, err := e.TSDBStore.SeriesCardinality(stmt.Database)
+	if err != nil {
+		return nil, err
+	}
+
+	return []*models.Row{&models.Row{
```

Comment:
Eurgh I realise this doesn't match the measurements case... Let's come to an agreement about what we want to output and I'll fix up the branch to do that for both commands.